admin-manual/installation/installation.rst

```diff
@@ -478,19 +533,40 @@ and enabled before the archivematica install. It can be done with:
 
    sudo rpm -Uvh https://forensics.cert.org/cert-forensics-tools-release-el7.rpm
 
-* Then, install the package:
+* Then install the package:
 
 .. code:: bash
 
    sudo -u root yum install -y archivematica-mcp-client
 
-* The MCP Client expect some programs in certain paths, so we put things in place:
+* The MCP Client expects some programs in certain paths, so we put them in place:
 
 .. code:: bash
 
-   sudo cp /usr/bin/clamscan /usr/bin/clamdscan
    sudo ln -s /usr/bin/7za /usr/bin/7z
 
+* Tweak ClamAV configuration:
+
+.. code:: bash
+
+   sudo -u root sed -i 's/^#TCPSocket/TCPSocket/g' /etc/clamd.d/scan.conf 
+   sudo -u root sed -i 's/^Example//g' /etc/clamd.d/scan.conf
+
+* Indexless mode:
+
+If you are planning to run Archivematica in indexless mode (i.e., without
+Elasticsearch), then set the relevant environment variables to ``false``
+
+If you are planning on running Archivematica in indexless mode (i.e., without
+Elasticsearch), then modify the relevant systemd EnvironmentFile files by
+adding lines that set the relevant environment variables to ``false``:
```

Comment:
A note like "Remember to stop elasticsearch if you don't want to use it, to free some resources" would be handy

Reply:
If the user is deploying an indexless Archivematica, then presumably they will have listened to the notes above this and not installed or started it in the first place. Therefore, I don't think that a note like you describe makes sense here. If you disagree, please let me know.